torch/_library/utils.py

```diff
@@ -96,3 +96,41 @@ def is_functional_schema(schema: Any) -> bool:
     if not schema.returns:
         return False
     return True
+
+
+def is_inplace_aten_op(op: torch._ops.OpOverload):
```

Comment:
Could we rename this to `mutates_and_returns_first_arg` ? I know we have an aten convention for inplace == mutating first arg but if this is also user fallbacks that may not be the case

Reply:
Sure. We won't apply this logic to custom ops, because the schema is not enough to tell us if that's what's going on. We have a convention in aten that schemas that look like this mean that the op mutates and returns the first arg.